USAGE.md

```diff
@@ -1,370 +1,707 @@
 
-### Get all Techniques
-Once the stix2 Python library is installed and the corpus is acquired, we need to open the DataStore for querying:
+Note that because groups use software and software uses techniques, groups can be considered indirect users of techniques used by their software. See [getting techniques used by a group's software](#Getting-techniques-used-by-a-groups-software), below.
+
+# Accessing ATT&CK Data in Python
+There are several ways to acquire the ATT&CK data in Python. All of them will provide an object 
+implementing the DataStore API and can be used interchangeably with the recipes provided in the [recipes](#Python-Recipes) section.
+
+
+## Access local content
+Many users may opt to access the ATT&CK content via a local copy of the STIX data on this repo. This can be advantageous for several reasons:
+- Doesn't require internet access after the initial download
+- User can modify the ATT&CK content if desired
+- Downloaded copy is static, so updates to the ATT&CK catalog won't cause bugs in automated workflows. User can still manually update by cloning a fresh version of the data
+
+### Access via FileSystemSource
+Each domain in this repo is formatted according to the [STIX2 FileSystem spec](https://stix2.readthedocs.io/en/latest/guide/filesystem.html). 
+Therefore you can use a `FileSystemSource` to load a domain, for example to load the enterprise-attack domain:
 
 ```python
-from stix2 import FileSystemSource
-fs = FileSystemSource('./cti/enterprise-attack')
+import stix2
+
+src = stix2.FileSystemSource('./cti/enterprise-attack')
 ```
 
-To perform a query, we must define a [Filter](http://stix2.readthedocs.io/en/latest/guide/datastore.html#Filters). As of this writing, a filter must, at a minimum, specify object `id`'s or an object `type`.  The following filter can be used to retrieve all ATT&CK techniques:
+### Access via Bundle
+If you instead prefer to download just the domain bundle, e.g [enterprise-attack.json](/enterprise-attack/enterprise-attack.json), you can still load this using a MemoryStore:
+```python
+import stix2
+
+src = stix2.MemoryStore()
+src.load_from_file("enterprise-attack.json")
+```
+
+## Access live content
+Some users may instead prefer to access "live" ATT&CK content over the internet. This is advantageous for several reasons:
+- Always stays up to date with the evolving ATT&CK catalog
+- Doesn't require an initial download of the ATT&CK content, generally requires less setup
+
+### Access from the ATT&CK TAXII server
+Users can access the ATT&CK data from the official ATT&CK TAXII server. In TAXII, the ATT&CK domains are represented as collections with static IDs:
```

Comment:
As a user, it would be valuable to have a link to the TAXII specification here. Or maybe to the TAXII intro page: https://oasis-open.github.io/cti-documentation/taxii/intro

And, is the TAXII server TAXII 2.0 or 2.1? That might be useful to include also.